src/cdk/activation.ts

```diff
@@ -82,16 +80,19 @@ async function registerCdkCommands(explorer: AwsCdkExplorer): Promise<void> {
         command: 'aws.cdk.provideFeedback',
         callback: async () => {
             vscode.env.openExternal(vscode.Uri.parse(cdkProvideFeedbackUrl))
-        }
+        },
+        telemetryName: 'Command_aws.cdkprovideFeedback'
     })
     registerCommand({
         command: 'aws.cdk.help',
         callback: async () => {
             vscode.env.openExternal(vscode.Uri.parse(cdkDocumentationUrl))
-        }
+        },
+        telemetryName: 'Command_aws.cdkhelp'
```

Comment:
This appears to be Command_aws.cdk.help in our metrics

Reply:
Fixed